docs/community/governance.md

```diff
@@ -0,0 +1,118 @@
+# GradleUp Governance Charter
+
+## What is GradleUp?
+
+GradleUp is a volunteer-driven umbrella GitHub organization
+focusing on simplifying developer experience with [Gradle Build Tool (a.k.a. Gradle)](https://gradle.org/),
+in particular for Java, Kotlin and Android projects.
+We host components and ensure their long-term sustainability.
+
+Our scope includes all things that help to achieve great user and developer experience when working with Gradle.
+It includes but not limited to plugins, developer tools, integrations, documentation or community content.
+
+## Formal Side
+
+The GradleUp organization is a volunteer driven community
+operating in a public interest in a non-profit mode and
+providing no commercial services.
```

Comment:
Should we explicit that there is no legal structure for GradleUp?

Reply:
Agreed

```suggestion
The GradleUp organization is a volunteer driven community
operating in a public interest in a non-profit mode and
providing no commercial services.
There is no formal legal entity behind GradleUp.
```